packages/peregrine/lib/talons/ProductFullDetail/useProductFullDetail.js

```diff
@@ -481,6 +497,12 @@ export const useProductFullDetail = props => {
 
                 try {
                     await addProductToCart({ variables });
+                    addToast({
+                        message: formatMessage({ defaultMessage: 'You added product to your shopping cart', id: 'productFullDetail.success'}, { name: product.name }),
+                        onDismiss: remove => remove(),
+                        timeout: 5000,
+                        type: 'success'
+                    });
```

Comment:
In the future, we suppose to do it in onCompleted() function of mutation to make sure toast is shown correctly only after adding a product to the cart.

But it's good.